src/backend/joanie/core/filters/admin.py

```diff
@@ -70,6 +74,44 @@ def filter_by_query(self, queryset, _name, value):
         """
         return queryset.filter(translations__title__icontains=value).distinct()
 
+    def filter_by_state(self, queryset, _name, value):
+        """
+        Filter resource by looking for states that match the one provided
+        in the "state" parameter
+        """
+        now = django_timezone.now()
+        state_texts = models.CourseState.STATE_TEXTS
+        if value == state_texts[models.CourseState.ONGOING_OPEN]:
```

Comment:
I think this is not going to work because texts are translatable...
If we want to filter on states we should just stick to integers or add a machine name for each state in the model definition if we think exposing integers sucks. 
For reminder we use integers in richie's search engine to reflect the ranking of course states in search results, which may be confusing when used outside of this context...